Application/EdFi.Ods.Api.IntegrationTestHarness/EdFi.Ods.Api.IntegrationTestHarness.csproj

```diff
@@ -32,7 +32,7 @@
     <PackageReference Include="Autofac" Version="6.3.0" />
     <PackageReference Include="Autofac.Extensions.DependencyInjection" Version="7.2.0" />
     <PackageReference Include="EdFi.Suite3.Admin.DataAccess" Version="6.0.12" />
-    <PackageReference Include="EdFi.Suite3.Security.DataAccess" Version="6.0.16" />
+    <PackageReference Include="EdFi.Suite3.Security.DataAccess" Version="6.0.18" />
```

Comment:
@semalaiappan I know this is a package that was already assigned and that you are updating the version, this package does not seem to be used by this project anywhere

Reply:
https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS-Implementation/blob/ODS-5543/Application/EdFi.Ods.Api.IntegrationTestHarness/PostmanSecurityMetadataInitializer.cs#L7